ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/base/v1/base/clientlibs/editor/js/editDialog.js

```diff
@@ -193,4 +286,52 @@
         }
     });
 
+    /**
+     * whenever RTE value of enumName is changed, we save it's value corresponding to text value field.
+     */
+    channel.on("change", "div[data-wrapperclass='cmp-adaptiveform-base__richTextEnumNames']", function (e) {
+        Coral.commons.ready(e.target, function(component) {
+            var i = $(component.parentNode.parentNode.parentNode.parentNode.parentNode).index(),
```

Comment:
This is not the right way. Please use CSS selectors. using parentNode would cause this to break easily as more fields in dialog gets added

Reply:
changed the approach